src/pollution_model/run_medusa_2.py

```diff
@@ -240,52 +252,59 @@ def total_metal_load_roof(surface_area: float,
     # Define error message (if needed)
     invalid_surface_error = (f"Given surface is not valid for computing total metal load."
                              f" Needed a roof, but got {SurfaceType(surface_type).name}.")
+
     # Define constants in a list
     match surface_type:
         case SurfaceType.COLOUR_STEEL:
-            b = [2, -2.802, 0.5, 0.217, 3.57, -0.09, 7, -3.732]
-            c = [910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.99]
+            # None was added to avoid the 0 position when slicing
+            b = [None, 2, -2.802, 0.5, 0.217, 3.57, -0.09, 7, -3.732]
+            c = [None, 910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.99]
             # Roof wash off coefficient (based on rate of decay to second concentrations from initial ones)
             k = 0.00933
             # Duration of the event measured by hours - Roof
             # observed from the intra-event concentration sampling
             z = 0.75
         case SurfaceType.GALVANISED:
-            b = [2, -2.802, 0.5, 0.217, 3.57, -0.09, 7, -3.732]
-            c = [910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.99]
+            # None was added to avoid the 0 position when slicing
+            b = [None, 2, -2.802, 0.5, 0.217, 3.57, -0.09, 7, -3.732]
+            c = [None, 910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.99]
             # Roof wash off coefficient (based on rate of decay to second concentrations from initial ones)
             k = 0.00933
             # Duration of the event measured by hours - Roof
             # observed from the intra-event concentration sampling
             z = 0.75
         case SurfaceType.METAL_OTHER:
+            # None was added to avoid the 0 position when slicing
             # Using coefficients of Galvanised rather than Copper
-            b = [2, -2.802, 0.5, 0.217, 3.57, -0.09, 7, -3.732]
-            c = [910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.99]
+            b = [None, 2, -2.802, 0.5, 0.217, 3.57, -0.09, 7, -3.732]
+            c = [None, 910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.99]
             # Roof wash off coefficient (based on rate of decay to second concentrations from initial ones)
             k = 0.00933
             # Duration of the event measured by hours - Roof
             # observed from the intra-event concentration sampling
             z = 0.75
         case SurfaceType.METAL_TILE:
-            b = [2, -2.802, 0.5, 0.217, 3.57, -0.09, 7, -3.732]
-            c = [910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.99]
+            # None was added to avoid the 0 position when slicing
```

Comment:
```suggestion
            # None was added to avoid the 0 position when indexing
```
Nitpick: This is called indexing, not slicing